components/inputs/input-time.js

```diff
@@ -241,23 +241,16 @@ class InputTime extends SkeletonMixin(FormElementMixin(LitElement)) {
 			this._formattedValue = formatTime(time);
 		}
 
+		const hiddenContent = this.shadowRoot.querySelector('.d2l-input-time-hidden-content');
 		this.addEventListener('d2l-localize-behavior-language-changed', () => {
 			this._formattedValue = formatTime(getDateFromISOTime(this.value));
 			INTERVALS.clear();
+			this.requestUpdate().then(() => this._onResize(hiddenContent));
```

Comment:
This is how `input-date` is working and would just ensure that the size change happens on language change as well as trigger the event for the tests.